toolchain/gnu/Makefile

```diff
@@ -0,0 +1,232 @@
+TOPDIR = $(CURDIR)
+
+include $(TOPDIR)/common.mk
+
+all: package
+
+build-host: gmp-install mpfr-install mpc-install isl-install cloog-install
+
+clean-host: gmp-clean mpfr-clean mpc-clean isl-clean cloog-clean
+	$(RM) -r $(HOSTDIR) $(HOSTBUILD)
+
+build-%-mimiker-elf: build-host binutils-unpack gcc-unpack gdb-unpack
+	$(MAKE) -C $*-mimiker-elf
+
+%-mimiker-elf_$(VERSION)_amd64.deb: build-%-mimiker-elf
+	$(MAKE) -C $*-mimiker-elf package
+
+clean-%-mimiker-elf:
+	$(MAKE) -C $*-mimiker-elf clean
+	$(RM) $*-.deb
+
+build: $(TARGETS:%=build-%-mimiker-elf)
+
+package: $(TARGETS:%=%-mimiker-elf_$(VERSION)_amd64.deb)
+
+clean: $(TARGETS:%=clean-%-mimiker-elf) clean-host
+	$(RM) -r sources
+
+.PHONY: all build clean build-host clean-host
+
+### GNU Binutils
+
+$(SRCDIR)/$(BINUTILS).tar.xz:
+	$(CURL) -o $@ $(BINUTILS-URL)
+
+binutils-unpack: $(SRCDIR)/binutils/.unpack
+$(SRCDIR)/binutils/.unpack: $(SRCDIR)/$(BINUTILS).tar.xz
+	cd $(SRCDIR) && tar xJf $^ && mv $(BINUTILS) binutils
```

Comment:
You shouldn't use any explicit commands like `cd`, `tar`, or `mv`. Appropriate wrappers should be used instead (please check mimiker/build/tools.mk).